apps/nuxt3-ssr/composables/useHeaderData.ts

```diff
@@ -0,0 +1,91 @@
+import type { IMgError } from "../interfaces/types";
```

Comment:
ah, this is nice to make it reusable. I still have to learn about these composables.